src/pages/projects/forms/ProjectDetailsForm.tsx

```diff
@@ -179,6 +181,34 @@ const ProjectDetailsForm: FC<Props> = ({ formik, project, isEdit }) => {
               ),
             }}
           />
+          <div>
+            <NetworkSelectorForm
+              value={formik.values.default_project_network}
+              setValue={(value) =>
+                void formik.setFieldValue("default_project_network", value)
+              }
+              selectProps={{
+                label: "Default network",
+                disabled:
+                  (formik.values.features_profiles === false &&
+                    features === "customised") ||
+                  isEdit,
```

Comment:
This should also disable, if features_networks is selected. Because then the project will not have access to the networks in the default project, so we should fall back to no network in the new project.